pkg/ddc/base/clean_up.go

```diff
@@ -0,0 +1,70 @@
+package base
+
+import (
+	"fmt"
+	"time"
+
+	datav1alpha1 "github.com/fluid-cloudnative/fluid/api/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// GetRemainingTimeToCleanUp return not nil remaining time if data operation has completion time and set ttlAfterFinished
+func GetRemainingTimeToCleanUp(object client.Object, opStatus *datav1alpha1.OperationStatus, operateType datav1alpha1.OperationType) (*time.Duration, error) {
```

Comment:
How about naming the function to Timeleft,  then put it into utils package.

Reply:
done.